be/src/storage/tablet.cpp

```diff
@@ -642,6 +650,7 @@ bool Tablet::add_committed_rowset(const RowsetSharedPtr& rowset) {
 }
 
 void Tablet::erase_committed_rowset(const RowsetSharedPtr& rowset) {
+    std::shared_lock l(_schema_lock);
```

Comment:
Should be `unique_lock` ?

Reply:
> Should be `unique_lock` ?

This lock is avoid concurrency between erase and `update_schema`, so i think shared_lock is enough.